app/src/main/java/ingsw/group1/findmyphone/location/LocationMessageHelper.java

```diff
@@ -3,23 +3,27 @@
 import android.location.Location;
 
 /**
- * Parser class of static methods that provides composition of message for location request and response
+ * Parser class of static methods that provides composition of message for location request and
+ * response
  * Message request is composed as:
  * {@link LocationMessageHelper#LOCATION_REQUEST_TAG}
  * <p>
  * Message response is composed in this way:
- * {@link LocationMessageHelper#LOCATION_RESPONSE_TAG}<LT>value_latitude</LT><LG>value_longitude</LG>
+ * {@link LocationMessageHelper#LOCATION_RESPONSE_TAG}<LT>value_latitude</LT><LG>value_longitude
+ * </LG>
  *
  * @author Giorgia Bortoletti
  */
 public class LocationMessageHelper {
 
-    protected static final String LOCATION_REQUEST_TAG = "LOCATION_REQUEST";
-    protected static final String LOCATION_RESPONSE_TAG = "LOCATION_RESPONSE";
-    private static final String LONGITUDE_TAG = "<LG>";
-    private static final String LONGITUDE_END_TAG = "</LG>";
-    private static final String LATITUDE_TAG = "<LT>";
-    private static final String LATITUDE_END_TAG = "</LT>";
+    static final String LOCATION_REQUEST_TAG = "LOCATION_REQUEST";
+    static final String LOCATION_RESPONSE_TAG = "LOCATION_RESPONSE";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=4950325)